clients/drcachesim/common/trace_entry.h

```diff
@@ -1039,4 +1062,22 @@ typedef enum {
  */
 #define DRMEMTRACE_CPU_SCHEDULE_FILENAME "cpu_schedule.bin.zip"
 
+/**
+ * The name of the folder in -offline mode where the kernel's per thread PT data is
+ * stored. This data is captured during online tracing.
```

Comment:
"online" is going to be confusing as "online" is used everywhere else here to mean the analysis tool is operating during tracing.  Best to delete "online" here and below.

Reply:
Done.